features/step_definitions/search_steps.rb

```diff
@@ -29,3 +31,26 @@
     page.should have_css('dt a span.match', text: text)
   end
 end
+
+And "the search input should be filled with {string}" do |query|
+  assert_equal query, find('input#search_query')[:value]
+end
+
+When "they search and there are no results" do
+  fill_in('search_query', with: 'FOO BAR BANANA')
+  click_button('Search')
+end
+
+Then "they should see an empty state" do
+  assert_equal 1, find_all('tbody tr').size
+  within('.pf-c-empty-state') do
+    assert has_css?('.pf-c-title', text: 'No results found')
+    assert has_css?('.pf-c-empty-state__body', text: I18n.t('buyers.accounts.empty_search.body'))
+  end
+end
+
+And "they should be able to reset the search" do
+  within('.pf-c-empty-state') do
+    click_link('Clear all filters')
+  end
```

Comment:
This is checking they can click the button, but it doesn't assert the button actually works, shouldn't it?

Reply:
You're right, I added more assertions. It's not perfect but still better.